dev/com.ibm.ws.grpc_fat/fat/src/com/ibm/ws/fat/grpc/FATSuite.java

```diff
@@ -39,4 +45,8 @@
 
     private static final Class<?> c = FATSuite.class;
 
+    @ClassRule
+    public static RepeatTests r = RepeatTests.withoutModification()
+                    .andWith(FeatureReplacementAction.EE9_FEATURES().alwaysAddFeature("servlet-5.0").fullFATOnly());
```

Comment:
Hmm. Maybe  I should remove servlet-4.0 and then add servlet-5.0 just to be safe.